### PR TITLE
draft/bugfix/1500 : display correct values when editing topic

### PR DESCRIPTION
**Pull request by jonasgeiregat**
**What changes did you make?** (Give an overview)

Made sure cleanup policy was displayed and updated correctly.
Aligned values for fetching and updating cleanup policy.
Displayed value for retention policy, this is not completely fixed yet.

**Is there anything you'd like reviewers to focus on?**

Before the retention policy was not displayed at all now it is but if you update it, it is not immediately reflected on the page. If you refresh or you go back and forth again it is correct. Maybe someone can point me in a direction what is going wrong here ?

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [ ] My changes generate no new warnings (e.g. Sonar is happy)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by Haarolean** [comment]
@jonasgeiregat thanks! Please look at the failing build, it seems like the linter is failing.

File: kafka-ui-react-app/src/components/Topics/shared/Form/TopicForm.tsx [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 import { StyledForm } from 'components/common/Form/Form.styled';
+import { TopicWithDetailedInfo } from 'redux/interfaces';
 
 import CustomParamsContainer from './CustomParams/CustomParamsContainer';
 import TimeToRetain from './TimeToRetain';
 import * as S from './TopicForm.styled';
 
 export interface Props {
-  topicName?: TopicName;
+  topic?: TopicWithDetailedInfo;
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Topics/shared/Form/TopicForm.tsx:18]: i think this component is used in another files as well so a change in its props name may effect those other places as well.

**Review by Mgrdich — CHANGES_REQUESTED** [review]
GitHub Actions / build-and-test warning should be fixed cause it makes pretty good suggestions.

**Comment by Mgrdich** [comment]
kafka-ui-react-app/src/components/Topics/List/ListItem.tsx this file is deleted , we should not include it during resolve.

**Review by lazzy-panda — COMMENTED** [review]
There are several linter warnings, please get rid of them.

**Pull request closed without merging**